### ISSUE TITLE
Readonly fields should still render inline errors and warnings

### ISSUE THREAD

**Issue by joshpencheon** [PublicHealthEngland/ndr_ui#12]
Fields that are rendered in readonly mode should still render the inline_errors_and_warnings.

```ruby
include ::NdrUi::Bootstrap::Readonly
include ::NdrUi::Bootstrap::TextField
include ::NdrUi::Bootstrap::InlineErrorsAndWarnings
```

Without looking in detail, the order of the includes looks sensible, so it may be a little tweak is needed somewhere.

**Comment by timgentry** [comment]
Hi @joshpencheon 997090b9308653ffef85790c506885b5ec9512a6 adds a test that shows that an error is shown when the form is readonly. Can you reproduce this issue or close? Thanks

**Comment by joshpencheon** [comment]
I've observed the test passing. I may have just been using a stale version of `ndr_ui` when I observed this, whilst checking against the latest version on GitHub. We can close this until I have evidence to the contrary!

**Comment by joshpencheon** [comment]
Found it - this is still an issue with `datepicker_field `.

**Comment by joshpencheon** [comment]
Closed by e4088f907.
